CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ CHANGES:
 * Bump go-tarantool from v2.3.0 to v2.3.1.
 * Get rid of nameToReplicasetMutex, use atomic instead.
 * Add configurable pause before retrying r.Route in Router.Call method.
+* Add ability to set custom dialer in InstaceInfo.
 
```

Comment:
I guess it would be better to squish all commits into a single one before merging.